src/pivot.coffee

```diff
@@ -112,6 +112,26 @@ callWithJQuery ($) ->
             format: formatter
             numInputs: if attr? then 0 else 1
 
+        sumMinusSum: (formatter=usFmt) -> ([num, denom]) -> (data, rowKey, colKey) ->
+            sumNum: 0
```

Comment:
Is it standard in this file to use `Num` instead of `Numerator`? `Num` just reads too close to "number" so I'd prefer to spell it out.

Reply:
That's how it was done for the ratio (sum over sum) aggregator. We can change it if you would like.